### ISSUE TITLE
Failed to build

### ISSUE THREAD

**Issue by Nardianian** [shakfu/sunvox-max#1]
this is my msys2mingw64 output
$ make
-- Selecting Windows SDK version 10.0.22621.0 to target Windows 10.0.22631.
-- The C compiler identification is MSVC 19.39.33520.0
-- The CXX compiler identification is MSVC 19.39.33520.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Generating: sunvox~
-- Configuring done (5.7s)
-- Generating done (0.1s)
-- Build files have been written to: C:/Users/ardian/source/repos/sunvox-max/build
make[1]: Entering directory '/c/Users/ardian/source/repos/sunvox-max/build'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory '/c/Users/ardian/source/repos/sunvox-max/build'
make: *** [Makefile:6: all] Error 2

**Comment by shakfu** [comment]
@Nardianian Thanks for your interest in this. 

It's not surprising that your build fails, as I previously was not been able to get sunvox to work in a Max external... hence the part in the README: "**Towards** an Max/MSP external which embeds the [sunvox](https://www.warmplace.ru/soft/sunvox/sunvox_lib.php) engine."

I initially implemented a [proof-of-concept](https://github.com/shakfu/pysunvox) but if I recall correctly the issue in this case was that there was some difficulty, at the time, to decouple the sound engine from sunvox and plug sunvox audio streams into the Max sound engine.

Nonetheless, after checking the sunvox documentation again just now, it looks there is actually a solution to this [here](https://warmplace.ru/soft/sunvox/sunvox_lib.php#sv_audio_callback)

So, in summary, it's currently non-functional, but there's some potential light at the end of the tunnel if one uses `sv_audio_callback` or `sv_audio_callback2`. I'll re-explore this line if time permits!

**Comment by Nardianian** [comment]
Thanks for your answer, i'm not a developer, i try but I hope this project is successful ...

**Comment by shakfu** [comment]
@Nardianian 

Appreciate your feedback.. it's always helpful.. let's see how where this `sv_audio_callback` goes :smile:

**Comment by shakfu** [comment]
@Nardianian  

I have a GitHub workflow that just successfully built a windows external and packaged it with a .dll. Would you be interested to test it?

**Comment by shakfu** [comment]
@Nardianian 

Can you please try the windows build in the [0.1.0 release](https://github.com/shakfu/sunvox-max/releases/tag/0.1.0)
